cyclus_input/predicting_the_past.xml

```diff
@@ -0,0 +1,183 @@
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>Enrichment1</name>
+    <config>
+      <Enrichment>
+        <feed_commod>uf6</feed_commod>
+        <feed_recipe>uf6</feed_recipe>
+        <product_commod>enr_u</product_commod>
+        <tails_commod>tails</tails_commod>
+        <max_enrich>0.045</max_enrich>
+      </Enrichment>
+    </config>
+  </facility>
+  <facility>
+    <name>FuelFab1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>enr_u</val>
+        </in_commods>
+        <out_commods>
+          <val>fresh_uox</val>
+        </out_commods>
+        <residence_time>1</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>WetStorage1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>spent_uox</val>
+        </in_commods>
+        <out_commods>
+          <val>cool_spent_uox</val>
+        </out_commods>
+        <residence_time>72</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>DryStorage1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>cool_spent_uox</val>
+        </in_commods>
+        <out_commods>
+          <val>casked_spent_uox</val>
+        </out_commods>
+      </Storage>
+    </config>
+  </facility>
+  <region>
+    <name>US</name>
+    <config>
+      <NullRegion/>
+    </config>
+    <institution>
+      <name>FuelCycle</name>
+      <initialfacilitylist>
+        <entry>
+          <prototype>Mine1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Mill1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Conversion1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Enrichment1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>FuelFab1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>WetStorage1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>DryStorage1</prototype>
+          <number>1</number>
+        </entry>
+      </initialfacilitylist>
+      <config>
+        <ManagerInst>
+          <prototypes>
+            <val>Mine1</val>
+            <val>Mill1</val>
+            <val>Conversion1</val>
+            <val>Enrichment1</val>
+            <val>FuelFab1</val>
+            <val>WetStorage1</val>
+            <val>DryStorage1</val>
+          </prototypes>
+        </ManagerInst>
+      </config>
+    </institution>
+    <institution>
+      <name>Power</name>
+      <config>
+        <DeployInst>
+          <xi:include href="./buildtimes/buildtimes.xml#xpointer(/simulation/DeployInsts/child::*)"/>
+        </DeployInst>
+      </config>
+    </institution>
+  </region>
+  <xi:include href="./recipes/uox_51.xml#xpointer(/recipes/child::*)"/>
```

Comment:
Unfortunately, no. I have tried to add the whole file without the <recipes> root node. However, without the recipes root note, the 'uox_51.xml' file, has two root nodes
'<recipe></recipe>
<recipe></recipe>'
Where the first recipe contains fresh fuel recipe, and the second recipe contains the spent fuel recipe but loading the whole file did not work as the parser recognizes the presence of the second recipe as a fragment.